book/src/controlling_system_execution/pausable_systems.md

```diff
@@ -0,0 +1,41 @@
+# Pausable Systems
+
+Custom `GameData` and state-specific `System`s are great when it comes to handling groups of `System`. But when it comes single `System`s or a group of `System`s spread over multiple `Dispatcher`s or `State`s, pausable `Sytem`s come in handy.
+
+Pausable `System`s can be enabled or disabled depending on the value of a`Resource` registered to your `World`. When this value changes, the state of your `System` changes as well.
+
+Let's get started by creating a new `Resource` that represents the state of our game.
+
+```rust,no_run,noplaypen
+#[derive(PartialEq)]
+pub enum CurrentState {
+    Running,
+    Paused,
+}
+
+impl Default for CurrentState {
+    fn default() -> Self {
+        CurrentState::Paused
+    }
+}
+```
+
+We'll use this `enum` `Resource` to control whether or not our `System` is running. Next we'll register our `System` and set it as pausable.
+
+```rust,no_run,noplaypen
+dispatcher.add(
+    MovementSystem::default().pausable(CurrentState::Running),
+    "movement_system",
+    &["input_system"],
+);
+```
+
+`pausable(CurrentState::Running)` creates a wrapper around our `System` that controls its execution depending on the `CurrentState` `Resource` registered with the `World`. As long as the value of the `Resource` is set to `CurrentState::Running`, the `System` is executed.
+
+To register the `Resource` or change its value, we can use the following line:
+
+```rust,no_run,noplaypen
+*data.world.write_resource::<CurrentState>() = CurrentState::Paused;
+```
+
+Alternatively we can access the `Resource` in `System`s utilising `amethyst::ecs::prelude::Read` and `amethyst::ecs::prelude::Write` as seen before. 
```

Comment:
This is not true for a system using the pausable extension. The Pausable wrapper actually accesses the resource with a 'Read' SystemData, so you cannot have a pausable system pause or unpause itself, as it causes a borrow error. The documentation should be clear on this that you *cannot* pause and unpause a system from within a pausable system, you need to lazy.exec the change. Or better yet, specifically state 'A Pausable system can only access its pause resource with immutable Read and cannot modify it' or something.

Reply:
Thanks for the feedback! I'll change that part tomorrow.